### ISSUE TITLE
[x86jit32] CscBench benchmark is broken.

### ISSUE THREAD

**Issue by jorive** [dotnet/coreclr#11088]
We have not got results since April 5th, 2017. A private PR performance run to make running benchmarks more robust indicates that `corerun.exe` is abnormally terminating when running the benchmark.

Benchmark: tests\src\JIT\Performance\CodeQuality\Roslyn\CscBench.cs

**Comment by gkhanna79** [comment]
CC @RussKeldorph

**Comment by RussKeldorph** [comment]
Just find out if this is bad enough that we need to fix for preview.

@dotnet/jit-contrib

**Comment by RussKeldorph** [comment]
```
>CscBench.cmd -coreroot c:\git\coreclr\bin\tests\Windows_NT.x86.Checked\Tests\Core_Root
BEGIN EXECUTION
 "c:\git\coreclr\bin\tests\Windows_NT.x86.Checked\Tests\Core_Root\corerun.exe" CscBench.exe

Assert failure(PID 18608 [0x000048b0], Thread: 17560 [0x4498]): Assertion failed '!"Unexpected fieldAccessor"' in 'Microsoft.Cci.BlobBuilder:.ctor(int):this'

    File: c:\git\coreclr\src\jit32\importer.cpp Line: 10930
    Image: c:\git\coreclr\bin\tests\Windows_NT.x86.Checked\Tests\Core_Root\CoreRun.exe

Expected: 100
Actual: 123456789
END EXECUTION - FAILED
FAILED
```
Looks like due to #9789.

**Comment by RussKeldorph** [comment]
This should be fixed in the release/2.0.0 branch.  No plan to keep JIT32 working in master.
